cielo/main.py

```diff
@@ -287,6 +302,30 @@ def get_authorized(self):
         self._authorized = True
         return True
 
+    def mask_sensible_info(
+        self, text, limit_start=LOGGING_DEFAULT_START_LIMIT, limit_end=LOGGING_DEFAULT_END_LIMIT,
+        replace_by="*"
+    ):
+        middle_lenght = len(text) - limit_start - limit_end
+        return "{}{}{}".format(text[:limit_start], replace_by*middle_lenght, text[-limit_end:])
+
+    def format_payload_for_logging(
+        self, payload, limit_start=LOGGING_DEFAULT_START_LIMIT, limit_end=LOGGING_DEFAULT_END_LIMIT
+    ):
+        payload = str(payload)
+        for xml_tag in LOGGING_MASK_XML_TAGS:
+            try:
+                info = re.search("<{0}>[A-Za-z0-9]*</{0}>".format(xml_tag), payload).group(0)
```

Comment:
Não existe a possibilidade de ter algum caractere diferente de `[A-Za-z0-9]` no valor?
Se existir a possibilidade, sugiro trocar para:
```suggestion
                info = re.search("<{0}>.*?</{0}>".format(xml_tag), payload).group(0)
```
Assim pega qualquer coisa que estiver aí no meio (e o `*?` pega o mínimo possível que satisfaça a regex).